modules/machineset-yaml-aws.adoc

```diff
@@ -0,0 +1,89 @@
+/ Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-aws.adoc
+
+[id="machineset-yaml-aws_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on AWS
+
+This sample YAML defines a MachineSet that runs in the `us-east-1a`
+Amazon Web Services (AWS) region and creates nodes that are labeled with
+`node-role.kubernetes.io/<role>: ""`
+
+In this sample, `<infrastructureID>` is the infrastructure ID label that is
+based on the cluster ID that you set when you provisioned
+the cluster, and `<role>` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+  name: <infrastructureID>-<role>-<zone> <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-us-east-1a <2>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <3>
+        machine.openshift.io/cluster-api-machine-type: <role> <3>
+        machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-us-east-1a <2>
```

Comment:
Should be consistent with machineset name, change this to `<infrastructureID>-<role>-<zone>` or change machineset name (line 24) to `<infrastructureID>-<role>-us-east-1a`